proc_scripts/plotting.py

```diff
@@ -105,17 +117,6 @@ def side_by_side(self,plotname,s,thisrange):
         return
     def plot_curve(fl, f, name, guess=None):
         """Plot the data with fit curve and fit equation.
-
-        Parameters
-        ----------
-        f: fitdata
-            data (a fitting instance), on which `f.fit()` has already been run
-        guess: None or nddata
-            The result of `s.settoguess();s.eval(100)` where 100 can be
-            any integer.
-            Used to display the guess on the plot as well.
-        name: str
-            the name of the plot
```

Comment:
You moved this up above... why?

Reply:
wasn't sure if the class needed parameters or if I put them under the functions of those classes. researched and saw like you say this goes under the function